docker/1.2-1/base/Dockerfile.cpu

```diff
@@ -29,6 +29,7 @@ RUN apt-get update && \
         wget \
         expat \
         tzdata \
+        apparmor\
```

Comment:
nit- do you need a space after the package name?

Reply:
oh this just escapes to the next line, but yeah for readability we had the space